README.md

```diff
@@ -98,6 +98,21 @@ If you are using jest-image-snapshot in an ephemeral environment (like a Continu
 To enable this image reporter, add it to your `jest.config.js` "reporters" definition:
 
     "reporters": [ "default", "<rootDir>/image-reporter.js" ]
+    
+ 
+#### Usage in TypeScript
+
+In TypeScript, you can declare the custom jest matcher `toMatchImageSnapshot` like this: 
```

Comment:
Can you remove the `customer jest matcher` bit? So it reads like:
`In TypeScript, you can declare `toMatchImageSnapshot` like this:`